### PR TITLE
Build people.json

### PR DESCRIPTION
**Pull request by La0**
Need some scopes to provide access to secrets from the hook
> requested https://phabricator.services.mozilla.com/D60230

### CODE REVIEW

File: bugzilla_dashboard/people.py [hunk 1]
```diff
@@ -38,129 +34,105 @@ def get_access_token(iam_credentials):
...
+            "id": user["id"],
+            "name": f"{profile['first_name']} {profile['last_name']}",
+            "email": profile["primary_email"],
+            "bugzillaEmail": profile["identities"][
+                "bugzilla_mozilla_org_primary_email"
+            ],
+            "manager": profile["access_information"]["hris"][
+                "managers_primary_work_email"
+            ],
+            "picture": profile["picture"],
```
Comment (calixteman) [inline bugzilla_dashboard/people.py]: Do we need that ?

Comment (La0) [reply bugzilla_dashboard/people.py]: We can display the avatar on the frontend, makes for a nicer interface

Comment (calixteman) [reply bugzilla_dashboard/people.py]: ok

**Comment by La0** [comment]
[It worked](https://firefox-ci-tc.services.mozilla.com/tasks/Nghh09TiQQKWDz3Ik-Sthw/runs/0#artifacts) :tada:

I need to update the hook artifact paths though.

**Pull request merged**